crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -431,11 +454,38 @@ public IEnumerable<View> GenerateViews()
 
     public record struct Constraint(ColumnDeclaration col, int pos, ColumnAttrs attr);
 
-    public IEnumerable<Constraint> GetConstraints(string viewName) =>
+    public IEnumerable<Constraint> GetConstraints(
+        TableView view,
+        ColumnAttrs filterByAttr = ~ColumnAttrs.UnSet
+    ) =>
         Members
             // Important: the position must be stored here, before filtering.
-            .Select((col, pos) => new Constraint(col, pos, col.GetAttrs(viewName)))
-            .Where(c => c.attr != ColumnAttrs.UnSet);
+            .Select((col, pos) => new Constraint(col, pos, col.GetAttrs(view)))
+            .Where(c => c.attr.HasFlag(filterByAttr));
+
+    public IEnumerable<ViewIndex> GetIndexes(TableView view) =>
+        Indexes.Where(i => i.Table == null || i.Table == view.Name);
+
+    // Reimplementation of V8 -> V9 constraint conversion in Rust.
```

Comment:
I've ripped most of this logic out in Rust modules FYI. They no longer have a notion of the "constraints" bitfield at all. The v9 format doesn't need it. In particular you don't have to generate indexes for unique constraints, the v9 validation code will do that while parsing; unless you want to control the names of the generated indexes. (Which I was TRYING to completely standardize, but alas, overruled.)

Reply:
Ah nice, happy to remove it here.

Reply:
But wait, V9 still has `Constraints` and `Sequences` collections, so this particular function is still needed.

It's only conversion to indexes that I can remove.

Reply:
Looks like that breaks due to accessor name comparisons in "ponder auto diff" function, which now fail because auto-generated accessor names are not the same.

I guess that's what you meant by

> unless you want to control the names of the generated indexes. (Which I was TRYING to completely standardize, but alas, overruled.)

but I think autogenerated accessor names are the ones that are wrong in this case?

Reply:
I'm going to just revert this change for now, but I think we might need to fix autogeneration code regardless.

Reply:
That's fine. It can be a followup PR to pull this out and double check the names are standardized between C# and rust.